src/theory/strings/regexp_solver.h

```diff
@@ -0,0 +1,102 @@
+/*********************                                                        */
+/*! \file regexp_solver.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Regular expression solver for the theory of strings.
+ **
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef __CVC4__THEORY__STRINGS__REGEXP_SOLVER_H
+#define __CVC4__THEORY__STRINGS__REGEXP_SOLVER_H
+
+#include "context/cdhashset.h"
+#include "context/cdlist.h"
+#include "theory/strings/regexp_operation.h"
```

Comment:
Missing `context/cdhashmap.h`, `context/context.h`, `expr/node.h`, `util/regexp.h`, `<map>`

Reply:
`<map>` goes before the other includes :)